src/Workspaces/Core/Portable/CodeActions/CodeActionOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.CodeActions
+{
+    /// <summary>
+    /// Options available to code fixes that are supplied by the IDE (i.e. not stored in editorconfig).
+    /// </summary>
+    internal readonly record struct CodeActionOptions(
+        bool IsBlocking,
+        bool SearchReferenceAssemblies,
+        bool HideAdvancedMembers)
```

Comment:
should this just reference AddImportOptions?  or is this the preferred way to go?

Reply:
It could but the reason I didn't do that is because `HideAdvancedMembers` option is also consumed by other code actions than AddImportsOptions. It'd be odd for these to access it via CodeActionOptions.AddImportsOptions.HideAdvancedMembers.